src/_css/map.css

```diff
@@ -37,3 +37,7 @@ canvas,
 .leaflet-control-container a:focus {
 	background-color: lightgray;
 }
+
+.leaflet-icon {
```

Comment:
We need this only for the map marker icons specifically. The others should still get rendered regularly (non-pixelated)

Reply:
This is a custom css class, but I renamed it now for clarity